packages/frontend/src/contexts/info.js

```diff
@@ -22,14 +22,18 @@ export default class Info {
 
   async load() {
     const url = new URL('api/info', SERVER)
-    const response = await fetch(url.toString())
+    const response = await fetch(url.toString(), {
+      method: 'get',
+      headers: { network: 'arbitrumGoerli' },
```

Comment:
Why specify the network as `arbitrumGoerli`?

Reply:
load network info from parameters
[cda06cd](https://github.com/Unirep/explorer/pull/68/commits/cda06cd377598bb7f04f3f2ee2a7181391c3c3d3)